drupal/values.yaml

```diff
@@ -482,9 +482,8 @@ gdprDump:
           parameters:
             key: 'processed_data.mail'
         name:
-          converter: 'faker'
-          parameters:
-            formatter: 'name'
+          converter: 'randomizeText'
```

Comment:
Comment from the sideline: Wuoldn't it be even simpler to use something like `user-${userId}`

If you want something nice-looking you could even use faker with name + append user id to ensure uniqueness

Reply:
You mean something like this?

```yml
        name:
          converter: faker
          parameters:
            formatter: name
            unique_prefix: '{{uid}}-'
```

Reply:
I guess so! Worth a try?